h/templates/admin/groups.html.jinja2

```diff
@@ -18,6 +18,7 @@
         <tr>
           <th>Name</th>
           <th>URL</th>
+          <th>Organization</th>
```

Comment:
Thanks for adding this here! Reminds me I wanted to add `type`, too (outside the scope of this PR, of course).